test/simulator-e2e-specs.js

```diff
@@ -360,6 +361,74 @@ function runTests (deviceType) {
       await getTouchEnrollKeys().should.eventually.deep.equal(originalValues);
     });
   });
+
+  describe(`multiple instances of ${deviceType.version} simulator on Xcode9+`, function () {
+    this.timeout(LONG_TIMEOUT);
+    const simulatorsMapping = new Map();
```

Comment:
because I use the _clear_ method, which plain object doesn't have. Also i find this data type more specialised for such type of tasks. Objects are too generic.